frontend/src/helpers/validation/index.ts

```diff
@@ -0,0 +1,2 @@
+export { validationEditUserSchema } from 'healthcare-shared/helpers';
```

Comment:
```suggestion
export { validationEditUser } from 'healthcare-shared/validation-schemas';
```
Move it from here and place in the right place